src/pages/settings/PasswordPage.js

```diff
@@ -101,6 +154,10 @@ class PasswordPage extends Component {
             });
     }
 
+    /**
+     * Check if new password matches confirm password field
+     * @returns {Boolean}
+     */
     doPasswordsMatch() {
```

Comment:
This method is kind of useless since it's only used once. We should just use something like `_.isEqual()`.